<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>COM Reader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../propolsion-ui/source/YwY2.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>串口检测</string>
            </property>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="1" column="0" colspan="2">
              <widget class="QLineEdit" name="com_show"/>
             </item>
             <item row="0" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>点火数据记录</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_start">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>开始</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_stop">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>停止</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_save">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>保存数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_clear">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>全部清空</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_figure">
          <property name="title">
           <string>F-t 图像</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3"/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_output">
        <property name="title">
         <string>端口数据监视</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QTextBrowser" name="textBrowser_info"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
